cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/0-intro.html

```diff
@@ -1,22 +1,36 @@
 <div class="college-costs_intro-segment">
-    {% for block in page.header -%}
-        {% if 'hero' in block.block_type %}
-            {{ render_stream_child(block) }}
-        {% endif %}
-    {%- endfor %}
-    <div class="wrapper content_wrapper">
-        <div class="content_main">
-           {% for block in page.header -%}
-                {% if block.block_type != 'hero' %}
+    {% if returning_user %}
+        <div class="wrapper content_wrapper">
+            <div class="content_main">
+                <div class="m-notification
+                            m-notification__visible">
+                    {{ svg_icon('update') }}
+                    <div class="m-notification_content">
+                        <div class="h4 m-notification_message">The page is loading…</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    {% else %}
+        {% for block in page.header -%}
+            {% if 'hero' in block.block_type %}
+                {{ render_stream_child(block) }}
+            {% endif %}
+        {%- endfor %}
+        <div class="wrapper content_wrapper">
+            <div class="content_main">
+               {% for block in page.header -%}
+                    {% if block.block_type != 'hero' %}
+                        {{ render_block.render(block, loop.index) }}
+                    {% endif %}
+                {%- endfor %}
+                {% for block in page.content -%}
                     {{ render_block.render(block, loop.index) }}
-                {% endif %}
-            {%- endfor %}
-            {% for block in page.content -%}
-                {{ render_block.render(block, loop.index) }}
-            {%- endfor %}
-            <div class="college-costs_tool-section_buttons">
-                <button class="a-btn college-costs_buttons__start btn__get-started">Get started</button>
+                {%- endfor %}
+                <div class="college-costs_tool-section_buttons">
+                    <button class="a-btn college-costs_buttons__start btn__get-started">Get started</button>
```

Comment:
Is `btn__get-started` a modifier on `a-btn`? If so it should be `a-btn__get-started`

Reply:
@anselmbradford It's just there as a JS selector, not a modifier.

Reply:
Hmm ok. We may want to treat it differently then so it doesn't seem like there could be styles attached. We have two precedents for this that I'm aware of:

- `js-` prefixing (e.g. `js-btn__get-started`), as is done [in YES](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/v1/youth_employment_success/route/notifications.html#L13).
- Using the [`data-js-hook` data attribute](https://github.com/cfpb/design-system/blob/main/packages/cfpb-atomic-component/src/utilities/standard-type.js), in particular the "behavior" type, if it meets [this definition](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/js/modules/util/behavior.js#L3-L20).